README.md

```diff
@@ -25,9 +25,13 @@ are gladly accepted!
 Setup
 =====
 
-Make sure you have jekyll installed (`gem install jekyll`), and run:
+Install the required dependencies using [Bundler](https://github.com/bundler/bundler)
 
-    $ jekyll serve
+    $ bundle install
```

Comment:
I think we'll need to execute `gem install bundler` before this (or
another installation method).

Reply:
Bundler is now shipping with RubyGems so it's not longer really necessary anymore.

Reply:
To be honest I'm still using 2.6, having a few issues updating to 2.7
(working on patches), but even when the official package for my OS
will be updated, I'm not sure yet the maintainers will keep Bundler in
the RubyGems package.

So I guess it might depends on your OS or distribution. Anyway it's
not that important, please disregard my comment then :-)